src/Symfony/Component/RemoteEvent/Event.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\RemoteEvent;
+
+class Event
+{
+    public function __construct(
+        private string $name,
+        private string $id,
+        private array $payload,
```

Comment:
Why not `public readonly` without getters instead?

Reply:
A public property is more demanding than a setter in terms of backward compatibility as it makes it harder to write a BC layer for it.